src/main/java/BankingSystem.java

```diff
@@ -0,0 +1,209 @@
+import dao.AccountDao;
+import model.Account;
+import model.Card;
+import model.CardGenerator;
+
+import java.util.Scanner;
+
+public class BankingSystem {
+
+    private final AccountDao dao;
+    private final CardGenerator cardGenerator;
+    private final Scanner scanner;
+    private Account currentAccount;
+
+    public BankingSystem(AccountDao dao) {
+        this.dao = dao;
+        cardGenerator = new CardGenerator();
+        scanner = new Scanner(System.in);
+    }
+
+    public void showMenu() {
+        while (true) {
+            welcomeMenu();
+
+            String input = scanner.next();
+
+            switch (input) {
+                case "0" -> {
+                    scanner.close();
+                    System.exit(0);
```

Comment:
It should be the same as in lines 93-96. Can you move it to the common method?

Reply:
Fixed